learn/framework/xcm.md

```diff
@@ -0,0 +1,73 @@
+
+Moving digital assets from one appchain to another is essential for creating a more dynamic, efficient, and interconnected blockchain ecosystem. The native cross-chain capability allows two main strategies to transfer assets from one chain to another:
+
+- **Teleport** - teleporting an asset is a simple and efficient mechanism, but it has a major caveat: it requires trust between the parties. In essence, when appchain A wants to send X amount of the assets to appchain B, it sends a message communicating appchain B that they must mint exactly X amount of the assets. After relaying the message, appchain A burns X amount of the assets, keeping the overall asset balance and concluding the teleport action
+- **Reserve transfer** - a reserve transfer involves a trusted third party acting as a reserve of value. When appchain A wants to send X amount of an asset to appchain B, it sends the instruction to the reserve, who transfers the assets from appchain A to appchain B reserve accounts. The process is completed when appchain A burns X amount of the assets and appchain B mints them. Note that regardless of appchain A not burning the assets or appchain B minting tokens in excess, since the wrongful tokens are not backed one-to-one by the reserve, they have no real value
+
+### Remote Execution {: #remote-execution }
+
+Another common use is remote execution, which is sending a message to an appchain and triggering some action. If the destination chain is an EVM-compatible one, this allows appchain A to call a smart contract deployed on appchain B, for example.
+
+## Establishing Cross-Chain Communication {: #channel-registration }
+
+Before two chains can start communicating, a messaging channel must be opened. Channels are unidirectional, meaning a channel from chain A to chain B will only pass messages from A to B. Therefore, two channels must be opened to send messages back and forth.
+
+A channel between the appchain and the relay chain is automatically opened upon appchain registration and onboarding. However, when appchain A wants to open a communication channel with appchain B, appchain A must send an open channel transaction to its network. This transaction is an XCM message as well!
+
+It's important to note that appchain A expressing its intentions of opening an XCM channel with appchain B is not sufficient. Appchain B must also accept the request by sending another XCM message. Only when both appchains have reached a mutual agreement, the channel is opened within the following epoch. The same process must be repeated to open a channel from appchain B to appchain A.
+
+![XCM Channel Registration Overview](/images/learn/framework/xcm/dark-xcm-1.webp#only-dark)
+![XCM Channel Registration Overview](/images/learn/framework/xcm/light-xcm-1.webp#only-dark#only-light)
+
+Once the channel is established, cross-chain messages can be sent between appchains. For asset transfers, assets will also need to be registered before being transferred.
```

Comment:
We never mention here also that the first XCM message sent between networks is to subscribe to the XCM version being used by the other chain. So that when the XCM version changes all the chains that there are channels opened should get notified of the version change